.github/workflows/validate-pull-requests.yml

```diff
@@ -14,8 +14,19 @@ jobs:
         uses: actions/setup-dotnet@v3.0.3
         with:
           dotnet-version: 7.0.x
+      - name: Check docstool
+        run: |
+           curl https://www.myget.org/F/particular/api/v3/flatcontainer/particular.docstool/index.json >> docstool.version
+           echo "/docstool" >> $env:GITHUB_PATH
+      - name: Cache docstool
+        id: cache-docstool
+        uses: actions/cache@v3
+        with:
+          path: /docstool
+          key: ${{ runner.os }}-docstool-${{ hashFiles('docstool.version') }}
```

Comment:
I don't think the runner os is really necessary. Should I just use docstool-hash?

Reply:
Yeah as long as we only run the workflow on one platform it shouldn't matter.

Though the cache keys are a repo-wide thing that would be shared by multiple workflows.

Reply:
Yeah they are but that should be a problem. I assume populating the cache is idempotent. So we even get bigger caching benefits